### ISSUE TITLE
NPE reported

### ISSUE THREAD

**Issue by dkarlinsky** [zio/zio-intellij#167]
Seeing the following NPE reported, in the zio project when editing `ZIOSpec`.
Tried enabling "show implicit hints" - not sure if related, but it had no effect for `ZIOSpec`, while working in other scala files.

```
2020-10-04 13:04:39,444 [264773913]  ERROR - aemon.impl.PassExecutorService - IntelliJ IDEA 2020.1.4  Build #IU-201.8743.12 
2020-10-04 13:04:39,444 [264773913]  ERROR - aemon.impl.PassExecutorService - JDK: 11.0.7; VM: OpenJDK 64-Bit Server VM; Vendor: JetBrains s.r.o. 
2020-10-04 13:04:39,444 [264773913]  ERROR - aemon.impl.PassExecutorService - OS: Mac OS X 
2020-10-04 13:04:39,444 [264773913]  ERROR - aemon.impl.PassExecutorService - Plugin to blame: ZIO for IntelliJ version: 2020.1.1.1 
2020-10-04 13:04:39,445 [264773914]  ERROR - aemon.impl.PassExecutorService - Last Action: ShowIntentionActions 
2020-10-04 13:04:39,445 [264773914]  ERROR - aemon.impl.PassExecutorService - null 
java.lang.NullPointerException
	at zio.intellij.inspections.package$StaticMemberReferenceExtractor.unapply(package.scala:191)
	at zio.intellij.inspections.package$StaticMemberReferenceExtractor.unapply$(package.scala:188)
	at zio.intellij.inspections.package$ZIOStaticMemberReferenceExtractor$.unapply(package.scala:197)
	at zio.intellij.inspections.package$StaticMemberReference.unapply(package.scala:129)
	at zio.intellij.inspections.mistakes.WrapInsteadOfLiftInspection$ExpressionExtractor.unapply(WrapInsteadOfLiftInspection.scala:76)
	at zio.intellij.inspections.mistakes.WrapInsteadOfLiftInspection.problemDescriptor(WrapInsteadOfLiftInspection.scala:57)
	at org.jetbrains.plugins.scala.codeInspection.AbstractRegisteredInspection$$anon$1.visitElement(AbstractRegisteredInspection.scala:26)
	at com.intellij.psi.impl.PsiElementBase.accept(PsiElementBase.java:273)
	at org.jetbrains.plugins.scala.lang.psi.impl.ScalaPsiElementImpl.org$jetbrains$plugins$scala$lang$psi$api$ScalaPsiElement$$super$accept(ScalaPsiElementImpl.scala:18)
	at org.jetbrains.plugins.scala.lang.psi.api.ScalaPsiElement.accept(ScalaPsiElement.scala:100)
	at org.jetbrains.plugins.scala.lang.psi.api.ScalaPsiElement.accept$(ScalaPsiElement.scala:98)
	at org.jetbrains.plugins.scala.lang.psi.impl.ScalaPsiElementImpl.accept(ScalaPsiElementImpl.scala:18)
	at com.intellij.codeInspection.InspectionEngine.acceptElements(InspectionEngine.java:69)
	at com.intellij.codeInsight.daemon.impl.LocalInspectionsPass.lambda$null$6(LocalInspectionsPass.java:320)
	at com.intellij.util.AstLoadingFilter.lambda$toComputable$2(AstLoadingFilter.java:168)
	at com.intellij.util.AstLoadingFilter.disallowTreeLoading(AstLoadingFilter.java:126)
	at com.intellij.util.AstLoadingFilter.disallowTreeLoading(AstLoadingFilter.java:115)
	at com.intellij.util.AstLoadingFilter.disallowTreeLoading(AstLoadingFilter.java:110)
	at com.intellij.codeInsight.daemon.impl.LocalInspectionsPass.lambda$visitRestElementsAndCleanup$7(LocalInspectionsPass.java:320)
	at com.intellij.concurrency.ApplierCompleter.execAndForkSubTasks(ApplierCompleter.java:149)
	at com.intellij.concurrency.ApplierCompleter.execAndForkSubTasks(ApplierCompleter.java:162)
	at com.intellij.openapi.application.impl.ApplicationImpl.tryRunReadAction(ApplicationImpl.java:1106)
	at com.intellij.concurrency.ApplierCompleter.lambda$wrapInReadActionAndIndicator$1(ApplierCompleter.java:105)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:627)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:572)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:61)
	at com.intellij.concurrency.ApplierCompleter.wrapInReadActionAndIndicator(ApplierCompleter.java:117)
	at com.intellij.concurrency.ApplierCompleter.lambda$compute$0(ApplierCompleter.java:96)
	at com.intellij.openapi.application.impl.ReadMostlyRWLock.executeByImpatientReader(ReadMostlyRWLock.java:168)
	at com.intellij.openapi.application.impl.ApplicationImpl.executeByImpatientReader(ApplicationImpl.java:168)
	at com.intellij.concurrency.ApplierCompleter.compute(ApplierCompleter.java:96)
	at java.base/java.util.concurrent.CountedCompleter.exec(CountedCompleter.java:746)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:290)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1020)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1656)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1594)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:177)
```

**Comment by hmemcpy** [comment]
Interesting. Thanks, will try to reproduce!

**Comment by timaliberdov** [comment]
@hmemcpy, it's probably `case f: ScFunctionDefinition if types.contains(f.containingClass.qualifiedName) => true` since containingClass can return null

**Comment by hmemcpy** [comment]
Haven't even looked, but makes sense. Gotta love unsafe API that can blow up at any moment!

**Comment by bertlebee** [comment]
I've also encountered this

**Comment by hmemcpy** [comment]
Closing this as it was (potentially) fixed in #173. If this happens again please re-open. Thanks!
